Student/Student/Submissions/SubmissionRubric/RubricViewController.swift

```diff
@@ -114,6 +132,7 @@ class RubricCollectionViewCell: UICollectionViewCell {
         circleView.rubric = rubric
         circleViewHeightConstraint.constant = RubricCircleView.computedHeight(rubric: rubric, maxWidth: bounds.size.width - (RubricCollectionViewCell.margin * 2))
         updateComment(comment: rubric.comment)
+        viewLongDescriptionButton.isHidden = rubric.longDescription.count == 0
```

Comment:
I'm guessing dave will say something about the extra white space this creates.  You may want to add this logic into the `computedHeight` method.

Reply:
👍 I will run it past him.